packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceRenameLightningComponent.test.ts

```diff
@@ -286,6 +286,8 @@ describe('Force Rename Lightning Component', () => {
       expect(isNameMatch(itemsInPage[6], componentName, componentPath)).to.equal(true);
       expect(isNameMatch(itemsInPage[7], componentName, componentPath)).to.equal(true);
       expect(isNameMatch(itemsInPage[8], componentName, componentPath)).to.equal(true);
+      expect(isNameMatch(itemsInPage[9], componentName, componentPath)).to.equal(true);
+      expect(isNameMatch(itemsInPage[10], componentName, componentPath)).to.equal(true);
     });
 
     it('should return false if file type is not in LWC or Aura or file name and component name do not match', () => {
```

Comment:
Can you update this test as well to add a negative test case like `handleEvt.js`. I could see it being common for people to name things with Evt.